clients/lcdproc/main.c

```diff
@@ -499,6 +516,7 @@ exit_program(int val)
 	 */
 	eyebox_clear();
 #endif
+	free(hostname);
```

Comment:
I wouldn't bother with freeing here. If you don't call free, you also don't need to strdup() the empty string above...

Reply:
But it is much nicer and elegant to clear used resources before exiting program. I know it doesn't change because program is beeing closed in a moment, but it is more professional.

This is also i've introduced another pull request for fixing detected leaks with valgrind.

Reply:
Well, C is often not nice or elegant, but practical ... :-)

I disagree about professionalism: I'm fine with adding extra code, if it helps readability (though comments are usually better), but in this case it only inflates the source and the footprint of the binary without any true benefit.